examples/pytorch/text-generation/train.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+$model_type=openai-gpt
```

Comment:
A minor syntax error. Please fix @HDThang

Reply:
Please note that our policy is to require at least one approval before merging into the main branch. Seems like we haven't fully reviewed these commits yet.

Reply:
The title of the PR #6 said this was for merging into branch `add_mul_choice_train_script`, not `main`. Idk somehow these changes still appear on `main` :shrug:

Do you have any idea why? @HDThang @huonglarne